Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance-WidgetExtension.xcscheme

```diff
@@ -90,7 +90,6 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES"
```

Comment:
Also fix this one!

Reply:
Same question as the other file am I actually fixing anything or should I just  'git ignore ' this file since you said to fix it.

Thanks for all your help!

Reply:
make sure to add this back